### PR TITLE
chore: add ydata bundle in pipelines

### PR DESCRIPTION
**Pull request by victoriavilasb**

### CODE REVIEW

**Review by rafaelcpalmeida — APPROVED** [review]

File: data-science/ydata/Makefile [hunk 1]
```diff
@@ -0,0 +1,32 @@
+TAG=latest
+DOCKER_REGISTRY=docker.pkg.github.com
```
Comment (portellaa) [inline data-science/ydata/Makefile:2]: this is no longer our docker registry we started migration already

**Review by portellaa — COMMENTED** [review]
please update the docker registry do the new model

File: data-science/ydata/Makefile (hunk 1 above)
Comment (victoriavilasb) [reply data-science/ydata/Makefile:2]: ok

File: data-science/ydata/Makefile [hunk 2]
```diff
@@ -0,0 +1,32 @@
+TAG=latest
+DOCKER_REGISTRY=ghcr.io
+PACKAGE=ydataai/dockerfiles
```
Comment (portellaa) [inline data-science/ydata/Makefile:3]: ghcr no longer supports packages specific to the project itself

**Review by portellaa — CHANGES_REQUESTED** [review]

**Review by portellaa — APPROVED** [review]

File: data-science/ydata/Makefile (hunk 2 above)
Comment (victoriavilasb) [reply data-science/ydata/Makefile]: right

**Pull request merged**